### DISCUSSION TITLE
Multi tenant

### DISCUSSION THREAD

**Discussion by mpie** [crater-invoice-inc/crater#573] [Ideas]
Would be cool if the app can be used by multiple companies at the same time without having to install multiple crater apps.

**Comment by DeanKamitsis** [comment]
I've been looking at the database structure for Crater, and it looks like it is something they are planning to introduce?

**Comment by mohitpanjwani** [comment]
Multiple companies are now supported in v5 👍

**Comment by onny** [comment]
real multi tanent support isn't planned for the free selfhosted version https://github.com/crater-invoice/crater/issues/35 😢
